src/scvi/external/decipher/utils/__init__.py

```diff
@@ -1,3 +1,4 @@
-from ._utils import rotate_decipher_components
+from ._rotate import rotate_decipher_components
```

Comment:
Im confused about the rotate file? It doesn't exist right?

Reply:
it exists within the utils folder